### ISSUE TITLE
NBT to PDC is still not working

### ISSUE THREAD

**Issue by Dezkrazzer** [GC-spigot/AdvancedEnchantments#4730]
### Describe the bug

So my server just updated to version 1.21 and obviously it has a lot of bugs especially with the AdvancedEnchanment book. The old books that were obtained before 1.21 can't be used at all, be it apply, tinkerer, or alchemist. Since there are so many players on my server and they keep the books, it's difficult for the staff to refund their books one by one. Is there any other solution or how to fix it?

### How to reproduce

Old enchant books obtained before the 1.21 update when applied to anything will not be able to be applied. The same goes for the book when in /tinkerer and also /alchemist.

### Screenshots / Videos

![image](https://github.com/user-attachments/assets/528a1e42-b0f0-4375-bd57-0d2436fe9828)

### "/ae plinfo" link

https://bin.hzmi.xyz/feyuwetesa.apache

### Filled out form correct and using latest version

I confirm

**Comment by GC-spigot** [comment]
We removed NBT in Spring, and had a grace period of 5+ months where you needed to update to latest plugin version which would automatically convert those items. Only recently NBT conversion was removed, when plugin was updated to 9.12.6 - October 25th. If you did not update in basically the half a year you had then the converter will not work anymore, as NBT handling is completely removed from our plugin. 

If your server version supports 9.12.5 plugin version, try updating to it. This will still convert items automatically. Otherwise there is nothing we can do.
